database/migrations/2020_03_06_000200_create_rock_the_vote_reports_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRockTheVoteReportsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('rock_the_vote_reports', function (Blueprint $table) {
+            $table->integer('id');
+            $table->string('status')->index();
+            $table->string('since')->nullable();
+            $table->string('before')->nullable();
+            $table->timestamps();
+            $table->date('imported_at')->nullable();
```

Comment:
I'm considering potentially renaming this field to `dispatched_at` -- it aims to save the time when we add a job to the queue to begin the import process (and wouldn't necessarily mean we imported the report if the download failed, for example)